correctionlib.spec

```diff
@@ -0,0 +1,18 @@
+### RPM external correctionlib 2.1.0
+## INITENV +PATH PYTHON27PATH %i/${PYTHON_LIB_SITE_PACKAGES}
+%define tag v%{realversion}cms0
```

Comment:
In future releases, the special Makefile will be in the regular release and we can drop the `cms0` suffix here.